### ISSUE TITLE
Not able to read Gitlab Repos

### ISSUE THREAD

**Issue by Yash621** [sourcebot-dev/sourcebot#97]
## Description

I am working on a project where i am trying to read repos through sourcebot, i am able to read github repos but even after repeated trials i am not able to read repos hosted on gitlab

## Example Config and Command

```
{
    "$schema": "https://raw.githubusercontent.com/sourcebot-dev/sourcebot/main/schemas/v2/index.json",
    "repos": [
        {
            "type": "gitlab",
            "url": "MY_GITLAB_URL",
            "token": {
                "env": "GITLAB_TOKEN"
            },
            "all": true,
            "groups": [
                "group-name"
            ]
        }
    ]
}
```

```
docker run -p 3000:3000 --rm --name sourcebot   -v $(pwd):/data   -e GITLAB_TOKEN=TOKEN_VALUE   -e CONFIG_PATH=/data/my_config.json   ghcr.io/sourcebot-dev/sourcebot:latest
```

this is the config and command that i am using in the project

**Comment by brendan-kellam** [comment]
Heya - could you enable debug logging by passing `-e SOURCEBOT_LOG_LEVEL=debug` with your `docker run` command and then send over your logs?

**Comment by Yash621** [comment]
these are the logs!

```

[Info] Sourcebot version: v2.2.0
[Info] Using config file at: '/data/my_config.json'.
2024-11-28 18:54:39,384 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2024-11-28 18:54:39,385 INFO supervisord started with pid 7
2024-11-28 18:54:40,390 INFO spawned: 'backend' with pid 132
2024-11-28 18:54:40,394 INFO spawned: 'web' with pid 133
2024-11-28 18:54:40,396 INFO spawned: 'zoekt' with pid 135
[zoekt] | 2024/11/28 18:54:40 loading 6 shard(s): github.com%2Fggerganov%2Fllama.cpp_v16.00000.zoekt... 1 more
[web] |   ▲ Next.js 14.2.10
[web] |   - Local:        http://localhost:3000
[web] |   - Network:      http://0.0.0.0:3000
[web] | 
[web] |  ✓ Starting...
[backend] | 2024-11-28T18:54:40.675Z info: [main] Syncing configuration file /data/my_config.json ...
[backend] | 2024-11-28T18:54:40.699Z debug: [GitLab] Fetching project info for group project...
[web] |  ✓ Ready in 92ms
2024-11-28 18:54:41,723 INFO success: backend entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-11-28 18:54:41,723 INFO success: web entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2024-11-28 18:54:41,723 INFO success: zoekt entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

```

this is the command i used !

```
docker run -p 3000:3000 --rm --name sourcebot   -v $(pwd):/data   -e GITLAB_TOKEN=GITLAB_TOKEN   -e CONFIG_PATH=/data/my_config.json -e SOURCEBOT_LOG_LEVEL=debug   ghcr.io/sourcebot-dev/sourcebot:latest
```

**Comment by brendan-kellam** [comment]
It looks like you are on a older version of Sourcebot (v2.2.0)- could you try updating to [latest](https://github.com/sourcebot-dev/sourcebot/releases/latest)? `docker pull ghcr.io/sourcebot-dev/sourcebot:latest`

I think the issue is that you are using the config option `all` which shipped in [v2.4.4](https://github.com/sourcebot-dev/sourcebot/releases/tag/v2.4.4)

**Comment by Yash621** [comment]
> It looks like you are on a older version of Sourcebot (v2.2.0)- could you try updating to [latest](https://github.com/sourcebot-dev/sourcebot/releases/latest)? `docker pull ghcr.io/sourcebot-dev/sourcebot:latest`
> 
> I think the issue is that you are using the config option `all` which shipped in [v2.4.4](https://github.com/sourcebot-dev/sourcebot/releases/tag/v2.4.4)

@brendan-kellam great it worked, thanks!
